src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs

```diff
@@ -78,5 +78,25 @@ public void ThenTheUserWillBeInformedTheLoginAttemptWasUnsuccessful(bool email,
                 _test.Driver.FindElement(By.CssSelector("[data-valmsg-for=Password]")).Displayed.Should().BeTrue();
             }
         }
+
+        [Given(@"the User is logged in")]
+        public void GivenTheUserIsLoggedIn()
+        {
+            GivenThatAUserIsNotLoggedIn();
+            WhenAUserProvidesRecognisedAuthenticationDetailsToLoginLocally();
+        }
+
+        [When(@"the User logs out")]
+        public void WhenTheUserLogsOut()
+        {
+            _test.Pages.Homepage.LogOut();
+        }
+
+        [Then(@"the User is logged out")]
+        public void ThenTheUserIsLoggedOut()
+        {
+            _test.Pages.Homepage.LoginLogoutLinkText().Should().Be("Log in");
```

Comment:
& here, does case matter?

Reply:
You are quite correct, fixed in next commit